src/team.iced

```diff
@@ -4,17 +4,62 @@
 {make_esc} = require 'iced-error'
 pgp_utils = require('pgp-utils')
 {json_stringify_sorted,unix_time,streq_secure} = pgp_utils.util
+{SubkeyBase} = require './subkey'
+{EncKeyManager,KeyManager} = require('kbpgp').kb
 
 #==========================================================================
 
-class TeamBase extends Base
+class TeamBase extends SubkeyBase
 
   constructor : (obj) ->
     @team = obj.team
+    @kms = obj.kms
     super obj
 
   _required_sections : () -> super().concat [ "team" ]
-  _v_customize_json : (ret) -> ret.body.team = @team
+  _v_customize_json : (ret) ->
+    ret.body.team = @team
+    if @per_team_key?
+      ret.body.team.per_team_key = @per_team_key
+
+  _v_generate : (opts, cb) ->
+    err = null
+    if @kms?
+      await super opts, defer err
+    cb err
+
+  get_field : () -> "per_team_key"
+  get_new_key_section : () -> @per_team_key
+  set_new_key_section : (m) ->
+    m.generation = @kms.generation
+    m.encryption_kid = @kms.encryption.get_ekid().toString('hex')
+    @per_team_key = m
+  get_new_km : () -> @kms?.signing # use the signing KM
```

Comment:
Does this mean that `helpers/key.iced` will do different things for the signing pubkey than it does for the encryption one?